src/rest.js

```diff
@@ -33,12 +32,15 @@ export class Rest {
    * @return {Promise}
    */
   request(method, path, body, options = {}) {
-    let requestOptions = extend(true, {}, this.defaults, options);
-
-    requestOptions.method = method;
-
-    if (typeof body === 'object') {
-      requestOptions.body = json(body);
+    let requestOptions = extend(true, {}, this.defaults, options, {method, body});
+
+    if (typeof body === 'object'
+    && !(typeof FormData === 'function' && body instanceof FormData)) {
```

Comment:
This indentation is not my liking. Combine the typeof and instanceof on body and put it all on one line. (You could _only_ use instance of, the typeof then becomes pointless)